### DISCUSSION TITLE
[Feature-Request] EPC-QR Code for Banking Data

### DISCUSSION THREAD

**Discussion by TheMBeat** [baimard/gestion#244] [Ideas]
I think an EPC-Code would a great addition.

An EPC QR code (also known as "GiroCode") is a QR code standardized by the European Payments Council (EPC) that contains all data for a SEPA transfer.

**Comment by baimard** [comment]
Sounds great link to documentation ?

**Comment by TheMBeat** [comment]
I think you could use this [npm Package](https://github.com/Bitfertig/epc-qr-code.js#readme) with MIT Licence.

**Comment by baimard** [comment]
Yes but documentation about girocode ?

**Comment by TheMBeat** [comment]
[The Documentation](https://www.europeanpaymentscouncil.eu/sites/default/files/kb/file/2018-05/EPC069-12%20v2.1%20Quick%20Response%20Code%20-%20Guidelines%20to%20Enable%20the%20Data%20Capture%20for%20the%20Initiation%20of%20a%20SCT.pdf)

**Comment by baimard** [comment]
thanks !

**Comment by baimard** [comment]
Planned
